operations/helm/charts/mimir-distributed/templates/admin-api/admin-api-servmon.yaml

```diff
@@ -5,9 +5,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "mimir.resourceName" (dict "ctx" $ "component" "admin-api") }}
-  {{- with .namespace }}
-  namespace: {{ . }}
-  {{- end }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
Actually here we may want to place this custom resource into the namespace of prometheus operator , and not into the release namespace, hence we used .Values.serviceMonitor.namespace, instead of the Release namespace. I actually have it running with my Mimir being in `dev` and Prom monitoring in the `monitoring` namespace:
```
serviceMonitor:
  enabled: true
  namespace: monitoring
  namespaceSelector:
    matchNames:
      - dev
```

Reply:
it might still make sense to default to the `.Release.Namespace`